src/bidiMapper/CommandProcessor.ts

```diff
@@ -271,7 +277,7 @@ export class CommandProcessor extends EventEmitter<CommandProcessorEvents> {
         this.emit(
           'response',
           OutgoingBidiMessage.createResolved(
-            new Message.UnknownErrorException(error).toErrorResponse(
+            new UnknownErrorException(error.toString()).toErrorResponse(
```

Comment:
This is incorrect we want to get the Error as it may have StackTraces we want to display.